### PR TITLE
Bug/rpm update#329#330

### PR DESCRIPTION
**Pull request by vgarciag**
This PR is about issues https://github.com/telefonicaid/fiware-connectors/issues/329 and  https://github.com/telefonicaid/fiware-connectors/issues/330.

Moreover an improvement has been made to make RPM package more fast letting the user to download flume form other repo using options -u and -a in package script.

Just execute flume/neore/script/package.sh -h to view all options

Note that you need to remove previous Cygnus package before install the new package. Then you can upgrade it without removing previous version

### CODE REVIEW

File: flume/neore/rpm/SPECS/cygnus.spec [hunk 1]
```diff
@@ -36,6 +36,9 @@ a Flume-based connector for context data coming from Orion Context Broker.
 %define _project_user cygnus
```
Comment (fgalan) [inline flume/neore/rpm/SPECS/cygnus.spec]: Should the fix of issues #329 and #330 be mentioned in the CHANGES_NEXT_RELEASE file?

Comment (vgarciag) [inline flume/neore/rpm/SPECS/cygnus.spec]: NTC

No, there aren't new features

**Comment by fgalan** [comment]
Can you elaborate on this, pls:

"Note that you need to remove previous Cygnus package before install the new package. Then you can upgrade it without removing previous version"

I mean, I don't understand why you need to "reset" removing the existing Cygnus before using RPM built with the new approach.

**Comment by vgarciag** [comment]
This is because previous package versions, before this PR, has the bug in which the upgrade of the package delete recent directories installation. If you don't uninstall first and next install a clean installation  of Cygnus the problem persists. So previous installed packages in the node is needed to be removed first.

To build new package is not needed to perform any special operation.

**Comment by fgalan** [comment]
Given that both #329 and #330 are scheduled in the 0.7.1 milestone, shouldn't this PR be merging into develop branch instend of release/0.6.0? Only critical issues (assigned to the IoTPv-1.1) merge into release/0.6.0.

(Sorry for commenting this now... I should have realize at the very beginning of seeing this PR :$)

**Comment by vgarciag** [comment]
I ask to @frbattid about this question and he told me that this fix will be merged in both 0.7.1 and 0.6.0, because 0.6.0 is active branch for IoT Platform 1.1. So I decided to open this branch from release/0.6.0 and do the PR in 0.6.0.

**Comment by fgalan** [comment]
Ok, I didn't know (or didn't remember :) that you have check that in advance.

Thus, LGTM (passing the ball to @frbattid for final LGTM + merge + twin PR in develop)

**Comment by frbattid** [comment]
@vgarciag said: <i>This is because previous package versions, before this PR, has the bug in which the upgrade of the package delete recent directories installation. If you don't uninstall first and next install a clean installation of Cygnus the problem persists. So previous installed packages in the node is needed to be removed first.</i>

If I've understood well, the uninstall step is only necessary if you have already installed a Cygnus version previous to 0.7.1/iotplatform-v1 (0.6.0). If not, this is not necessary (I mean, in a future 0.8.0 release this won't be necessary, right?).

I would add a comment in the README.md regarding this (RPM installation section).

**Comment by vgarciag** [comment]
All packages generated before this PR have the bug and is needed to be uninstalled first. Then, upgrades, installations or uninstallations should work well

**Comment by vgarciag** [comment]
A Note in [README.md](https://github.com/telefonicaid/fiware-connectors/tree/bug/rpm_update%23329%23330/flume/neore) has been added trying to explain how to solve this issue.

**Comment by frbattid** [comment]
LGTM

**Pull request merged**